docs/documentation/gui.md

```diff
@@ -89,12 +89,12 @@ To use the GUI, start by opening the app, select the form corresponding to the a
 - **String**: Accepts text strings.
 - **Restricted choice string**: Accepts predefined choices from a list.
 - **File path**: Allows selection of a file and inputs its absolute path.
-- **Boolean**: Accepts true/false values in shape of checkboxes.
+- **Boolean**: Accepts true/false values in the shape of checkboxes.
 - **Directory path**: Allows selection of a directory and inputs its absolute path.
 - **Dictionary**: Accepts key-value pairs.
-- **List of string**: Accepts a list of text strings.
-- **List of integer**: Accepts a list of integer values.
-- **List of file path**: Accepts a list of file paths.
+- **List of strings**: Accepts a list of text strings.
+- **List of integers**: Accepts a list of integer values.
+- **List of file paths**: Accepts a list of file paths.
 - **List of double precision**: Accepts a list of double precision floating point values.
```

Comment:
This list of field type is self explanatory. Adding a lengthy description is in most case completely unnecessary imho.